scanpipe/models.py

```diff
@@ -2292,6 +2292,15 @@ def file_content(self):
         for optimal compatibility.
         """
         from textcode.analysis import numbered_text_lines
+        from typecode import get_type
+
+        # When reading a map file, Textcode only provides the content inside
+        # `sourcesContent`, which can be misleading during any kind of review.
+        # This workaround ensures that the entire content of map files is displayed.
+        file_type = get_type(self.location)
+        if file_type.is_js_map:
+            f = open(self.location, "r")
```

Comment:
Use a context manager when opening files to ensure those are properly closed.

Reply:
Done :)